frontend-pwa/src/screens/SpeciesList.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Container, Typography, IconButton, Box, List, ListItem, ListItemText, ListItemAvatar, Avatar } from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import ArrowBackIcon from '@mui/icons-material/ArrowBack';
+
+// Mock data
```

Comment:
Don't use mock data, the api is available. Use the same endpoints as in 'frontend'.